### PR TITLE
index.php file for controllers/front/listing/

### PR DESCRIPTION
**Pull request by Seleda**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | index.php file for controllers/front/listing/
| Type?         | improvement
| Category?     | FO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | 
| How to test?  | there is no index.php file

#### Important guidelines

* Make sure [your local branch is up to date](https://help.github.com/articles/syncing-a-fork/) before commiting your changes!
* Your code MUST respect [our Coding Standards](http://doc.prestashop.com/display/PS16/Coding+Standards) (for code written in PHP, JavaScript, HTML/CSS/Smarty/Twig, SQL)!

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/9128)

### CODE REVIEW

File: controllers/front/ContactController.php [hunk 1]
```diff
@@ -28,6 +28,14 @@ class ContactControllerCore extends FrontController
...
     public $php_self = 'contact';
     public $ssl = true;
+    public $auth = true;
+    
+    public function __construct()
+    {
+        parent::__construct();
+        
+        $this->authRedirection = $this->context->link->getPageLink($this->php_self);
+    }
```
Comment (Quetzacoalt91) [inline controllers/front/ContactController.php]: I guess that change wasn't wanted in this PR?

Comment (Seleda) [reply controllers/front/ContactController.php]: I have thus defended some of the sites from bots that spread spam

Comment (Seleda) [reply controllers/front/ContactController.php]: I deleted this fragment

**Review by Quetzacoalt91 — APPROVED** [review]

**Comment by PierreRambaud** [comment]
@eternoendless @mickaelandrieu We maybe can merge this one without QA feedback no?

**Comment by Quetzacoalt91** [comment]
Indeed, we can merge it directly.

Thank you @Seleda

**Pull request merged**